src/Microsoft.AspNetCore.Razor.Tasks/RazorGenerate.cs

```diff
@@ -1,28 +1,44 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.IO;
 using System.Text;
 using Microsoft.Build.Framework;
-using Microsoft.CodeAnalysis.CommandLine;
 
 namespace Microsoft.AspNetCore.Razor.Tasks
 {
     public class RazorGenerate : DotNetToolTask
     {
-        [Required]
-        public string[] Sources { get; set; }
+        private const string GeneratedOutput = nameof(GeneratedOutput);
+        private const string TargetPath = nameof(TargetPath);
+        private const string FullPath = nameof(FullPath);
```

Comment:
Not a fan of this usage of nameof. The strings **can't** change. nameof does the opposite of what you want